flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/internal/MongoDBConnectorSourceTask.java

```diff
@@ -226,4 +256,103 @@ private boolean isCopying() {
             throw new IllegalStateException("Cannot access isCopying field of SourceTask", e);
         }
     }
+
+    private void init(Map<String, String> props) {
```

Comment:
```suggestion
    private void initCapturedCollections(MongoClient mongoClient, Map<String, String> props) {
```
and then we can put this method to `CollectionDiscoveryUtils`

Reply:
Fixed